packages/core/src/modules/proofs/services/ProofService.ts

```diff
@@ -1137,6 +1143,27 @@ export class ProofService {
       )
     }
   }
+
+  public assertAttributeNamesDoNotMatch(proofRequest: ProofRequest) {
+    const attributes = Array.from(proofRequest.requestedAttributes.values()).map((a) => a.name)
+    const duplicates = attributes.filter((item, index) => attributes.indexOf(item) != index)
+
+    if (duplicates.length > 0) {
+      throw new AriesFrameworkError(`The proof request contains a duplicate attribute name: ${duplicates}`)
```

Comment:
Hopefully soon we will add some stricter typing. But arrays will not be allowed anymore in template literals. Could you replace `${duplicates}` to `${duplicates.toString()}` at all occurrences?

Reply:
Yes!